### PR TITLE
Rename 'master' branch to 'main'

### PR DESCRIPTION
**Pull request by thomas-neitmann**

### CODE REVIEW

**Review by statasaurus — APPROVED** [review]
Looks great! Thanks for getting to this so quickly

**Comment by thomas-neitmann** [comment]
@statasaurus Could you approve once again? I had to fix a merge conflict. Unless you approve again the "Check Templates" workflow is not triggered.

**Pull request merged**
